x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/InitialNodeSecurityAutoConfiguration.java

```diff
@@ -191,91 +192,209 @@ protected void doRun() {
+                    + " Start Elasticsearch with "
+                    + cmdOn
+                    + "bin/elasticsearch --enrollment-token <token>"
+                    + cmdOff
+                    + ", using the enrollment token that you generated."
+            );
+        } else if (Strings.isEmpty(nodeEnrollmentToken)) {
+            builder.append(infoBullet + " Configure other nodes to join this cluster:");
+            builder.append(System.lineSeparator());
+            builder.append(bullet + " On this node:");
             builder.append(System.lineSeparator());
-            builder.append(caCertFingerprint);
+            builder.append(
+                "  "
+                    + hyphenBullet
+                    + " Create an enrollment token with "
+                    + cmdOn
+                    + "bin/elasticsearch-create-enrollment-token -s node"
+                    + cmdOff
+                    + "."
+            );
+            builder.append(System.lineSeparator());
+            builder.append(
+                "  "
+                    + hyphenBullet
+                    + " Uncomment the "
+                    + boldOnANSI
+                    + "transport.host"
+                    + boldOffANSI
+                    + " setting at the end of "
+                    + boldOnANSI
+                    + "config/elasticsearch.yml"
+                    + boldOffANSI
+                    + "."
+            );
+            builder.append(System.lineSeparator());
+            builder.append("  " + hyphenBullet + " Restart Elasticsearch.");
+            builder.append(System.lineSeparator());
+            builder.append(bullet + " On other nodes:");
+            builder.append(System.lineSeparator());
+            builder.append(
+                "  "
+                    + hyphenBullet
+                    + " Start Elasticsearch with "
+                    + cmdOn
+                    + "bin/elasticsearch --enrollment-token <token>"
+                    + cmdOff
+                    + ", using the enrollment token that you generated."
+            );
+        } else {
+            builder.append(infoBullet + " Configure other nodes to join this cluster:");
+            builder.append(System.lineSeparator());
+            builder.append(
+                bullet
+                    + " Copy the following enrollment token and start new Elasticsearch nodes with "
+                    + cmdOn
+                    + "bin/elasticsearch --enrollment-token <token>"
+                    + cmdOff
+                    + " (valid for the next 30 minutes):"
+            );
+            builder.append(System.lineSeparator());
+            builder.append("  " + boldOnANSI + nodeEnrollmentToken + boldOffANSI);
             builder.append(System.lineSeparator());
+            builder.append(System.lineSeparator());
+            builder.append("If you're running in Docker, copy the enrollment token and run:");
```

Comment:
let's add two spaces here so that it is clearer that it applies to the last step and is not an additional step on its own